android/MainActivity.kt

```
@@ -0,0 +1,1247 @@
+                filePickError = "No file selected"
+                filePickInFlight = false
+            }
+            return@registerForActivityResult
+        }
+        val mimeType = contentResolver.getType(uri) ?: "application/octet-stream"
+        val resolver = contentResolver
+        try {
+            Thread {
+                processPickedContent(uri, mimeType, "file", resolver)
+            }.start()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)